homeassistant/components/spotify/media_player.py

```diff
@@ -202,9 +203,9 @@ def media_position(self) -> int | None:
     @property
     def media_position_updated_at(self) -> dt.datetime | None:
         """When was the position of the current playing media valid."""
-        if not self._currently_playing:
+        if not self._currently_playing or self._currently_playing_updated is None:
```

Comment:
Why do we do `or`? When we aren't playing anything, we don't have a current playing updated right?